Makefile

```diff
@@ -0,0 +1,54 @@
+NAME=go-md2man
+BINARYNAME=md2man
+OWNER=cpuguy83
+ARCH=$(shell uname -m)
+VERSION=0.0.0
+
+build: build-darwin build-linux
```

Comment:
I ended up calling it `build-cross` to signify that it is still a `build` target of sorts.